func/SessionProcessor.cs

```diff
@@ -44,7 +46,20 @@ public static class SessionProcessor
 
         static SessionProcessor()
         {
-            var key = Environment.GetEnvironmentVariable("AzureOpenAI.Key");
+            var keyVaultEndpoint = Environment.GetEnvironmentVariable("AzureKeyVault.Endpoint");
```

Comment:
Could we switch this to use the capital and snake casing for environment variables like we do in the other samples?

Reply:
Updated.